src/Conversion/ONNXToKrnl/Math/Elementwise.cpp

```diff
@@ -725,6 +725,7 @@ struct ONNXElementwiseBinaryOpLowering : public ConversionPattern {
         NameLoc::get(Identifier::get(ElementwiseBinaryOp::getOperationName(),
                          op->getContext()),
             op->getLoc());
+    rewriter.create<KrnlInstrumentOp>(loc, op, 1);
```

Comment:
can we have an enum for the tag argument? then we could print something meaningful?